config/test.exs

```diff
@@ -22,7 +22,6 @@ config :code_corps, site_url: "http://localhost:4200"
 
 # speed up password hashing
 config :comeonin, :bcrypt_log_rounds, 4
-config :comeonin, :pbkdf2_rounds, 1
```

Comment:
Can you clarify why this is removed and what it changes?

Reply:
Oh I guess I didn't see it used anywhere.  `pbkdf2_rounds` specifically in the app anywhere.

Reply:
I think it's because there's a default that this overrides. Am I wrong on that? This just makes the password hashing faster.

Reply:
It's the recommended approach to speed up hashing in test environment only. However, since we use bcrypt and not pbkdf2, it's unnecessary.